ccdproc/combiner.py

```diff
@@ -329,6 +331,9 @@ def average_combine(self, scale_func=ma.average, scale_to=None):
                Scaling factor used in the average combined image. If given,
                it overrides ``CCDData.scaling``. Defaults to None.
 
+           uncertainty_func: function, optional
+                Function to calculate uncertainty. Defaults to numpy.ma.std
```

Comment:
Could you wrap `numpy.ma.std` with single backticks? That will tell sphinx to convert it to a link to the numpy documentation:
\`numpy.ma.std\`

Same also for the sigma-func.